lerna.json

```diff
@@ -0,0 +1,17 @@
+{
+    "command": {
+        "publish": {
+            "message": "chore(release): publish"
+        }
+    },
+    "ignoreChanges": [
+        "**/CHANGELOG.md",
+        "**/{storybook,stories}/**"
+    ],
+    "packages": [
+        "packages/*"
+    ],
+    "useWorkspaces": true,
+    "npmClient": "yarn",
+    "version": "independent"
```

Comment:
Not sure about this. I do not see us supporting `core` `v2` to working with say `content` `v3`. `v2` of any package will work with only `v2` of other packages.